{"uuid": "32571cf0-c6c9-49d7-8f5e-6e03682ebeb7", "children": ["0d4d9060-784f-4383-add2-8b4a913bb7e1"], "befores": [{"name": "admin_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'dict' object has no attribute 'stringify'\n", "trace": "  File \"/Users/zhouqi/PycharmProjects/pytestDemo1/venv/lib/python3.8/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/zhouqi/PycharmProjects/pytestDemo1/venv/lib/python3.8/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/zhouqi/PycharmProjects/pytestDemo1/venv/lib/python3.8/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/zhouqi/PycharmProjects/pytestDemo1/venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/zhouqi/PycharmProjects/pytestDemo1/venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/zhouqi/PycharmProjects/pytestDemo1/testcases/api_test/admin/conftest.py\", line 30, in admin_login\n    res_json = json.stringify(res)\n"}, "start": 1736683727229, "stop": 1736683727496}], "start": 1736683727229, "stop": 1736683727577}